docs/user_guide/assessment/index.rst

```diff
@@ -11,6 +11,13 @@ fairness metrics, such as demographic parity and equalized odds.
 We will show how :class:`MetricFrame` can be used to evaluate the metrics
 identified during the course of a fairness assessment.
 
+Fairlean provides two primary ways of assessing fairness: :class:`MetricFrame`,
+which can be used to perform disaggregated analysis of a particular performance 
```

Comment:
I'm fine with this. Technically, though, you can get the single number from MetricFrame, too. It's just that if you don't care about by-group values then there's no reason to use MetricFrame

Reply:
Added a sentence: " MetricFrame can also be used to output an aggregate number, but the predefined fairness metrics can be used when direct by-group comparison is not necessary." Let me know if that is on the right track.

Reply:
I know it's mentioned down below, but do you want to mention here that the predefined fairness metrics use `MetricFrame` internally?

Reply:
Sure, I have changed that sentence to say "and a set of predefined fairness metrics that use :class:`MetricFrame` internally to output an aggregate number."